server/app/views/applicant/ApplicantProgramBlockEditTemplate.html

```diff
@@ -0,0 +1,12 @@
+<html>
```

Comment:
**question**: This file doesn't start with `<!DOCTYPE html>`, but is it still required at the start of the file, in practice? Technically W3C says it is, but do browsers still even have a quirks mode anymore we'd need to steer them clear of?

Reply:
done